mongorestore/metadata.go

```diff
@@ -36,17 +36,16 @@ type authVersionPair struct {
 
 // Metadata holds information about a collection's options and indexes.
 type Metadata struct {
-	Options        bson.D          `bson:"options,omitempty"`
-	Indexes        []IndexDocument `bson:"indexes"`
-	UUID           string          `bson:"uuid"`
-	CollectionName string          `bson:"collectionName"`
+	Options        bson.D                  `bson:"options,omitempty"`
+	Indexes        []intents.IndexDocument `bson:"indexes"`
+	UUID           string                  `bson:"uuid"`
+	CollectionName string                  `bson:"collectionName"`
+	DatabaseName   string                  `bson:",omitempty"`
```

Comment:
where does this `DatabaseName` field get used? C-f isn't finding it for me in this patch

Reply:
Ah yeah, at some point during development I was using it but not anymore.